sql/src/test/java/org/opensearch/sql/sql/antlr/SQLSyntaxParserTest.java

```diff
@@ -194,6 +194,32 @@ public void can_parse_now_like_functions(String name, Boolean hasFsp, Boolean ha
     assertNotNull(parser.parse("SELECT id FROM test WHERE " + String.join(" AND ", calls)));
   }
 
+  private static Stream<Arguments> get_format_arguments() {
+    return Stream.of(
+        Arguments.of("GET_FORMAT(DATE,'USA')"),
+        Arguments.of("GET_FORMAT(DATE,'JIS')"),
+        Arguments.of("GET_FORMAT(DATE,'ISO')"),
+        Arguments.of("GET_FORMAT(DATE,'EUR')"),
+        Arguments.of("GET_FORMAT(DATE,'INTERNAL')"),
+        Arguments.of("GET_FORMAT(DATETIME,'USA')"),
+        Arguments.of("GET_FORMAT(DATETIME,'JIS')"),
+        Arguments.of("GET_FORMAT(DATETIME,'ISO')"),
+        Arguments.of("GET_FORMAT(DATETIME,'EUR')"),
+        Arguments.of("GET_FORMAT(DATETIME,'INTERNAL')"),
+        Arguments.of("GET_FORMAT(TIME,'USA')"),
+        Arguments.of("GET_FORMAT(TIME,'JIS')"),
+        Arguments.of("GET_FORMAT(TIME,'ISO')"),
+        Arguments.of("GET_FORMAT(TIME,'EUR')"),
+        Arguments.of("GET_FORMAT(TIME,'INTERNAL')")
+    );
+  }
+
+  @ParameterizedTest(name = "{0}")
+  @MethodSource("get_format_arguments")
+  public void can_parse_get_format_function(String getFormatQuery) {
```

Comment:
Test for incorrect input for first arg

Reply:
Added in https://github.com/Bit-Quill/opensearch-project-sql/pull/210/commits/aee2cb553fbc9307f3e0cb96b325e671b514da14